pkg/awsapi/ssm.go

```diff
@@ -11,29 +11,30 @@ import (
 // SSM provides an interface to the AWS SSM service.
```

Comment:
Hm, this stems from the update, right?

Reply:
Yeah, I couldn't figure out a way to `make build` without this getting changed, do you know if there's some env vars or a flag I should be using during `make build` to avoid these overwrites?

Reply:
Sadly, I don't. These can be annoying, indeed.